shell/deepdive-run

```diff
@@ -108,6 +108,9 @@ fullConfig=$run_dir/deepdive.conf
         echo 1
     ) 2>/dev/null)}
     export PARALLELISM
+    
+    # set INPUT_BATCH_SIZE env var 
+    export INPUT_BATCH_SIZE=10000
```

Comment:
Always settings this prevents the user from overriding the value.  Please fix it to respect user's environment.